src/components/embed_pages/FaceToFaceClassInfoEmbedPage.ts

```diff
@@ -0,0 +1,84 @@
+import { EmbedPage } from '../models/EmbedPage';
+import { client } from '../../index';
+import * as event from '../../../assets/event.json';
+import { AlgoBot } from '../../client/Client';
+
+export const data = new EmbedPage(
+    client,
+    false,
+    false,
+    false,
+    'nextFaceToFaceClass',
+    'Información de la próxima clase presencial',
+    '',
+    [client.config.generalTextChannelID],
+    [
+        {
+            name: 'Aula',
+            value: 'undefined',
+        },
+        {
+            name: 'Detalles de la clase',
+            value: `Empieza el ${AlgoBot.dateFromISO(
+                event.start.dateTime,
+                event.start.timeZone
+            )} y termina el ${AlgoBot.dateFromISO(
+                event.end.dateTime,
+                event.end.timeZone
+            )}`,
+        },
+        {
+            name: 'Temas a ver',
+            value: `${extractTopicsFromEventDescription(event.description)}`,
+        },
+        {
+            name: 'Ejercicio a tener entregado',
+            value: `${extractExercisesFromEventDescription(event.description)}`,
+        },
+        {
+            name: 'Paper a tener leído',
+            value: `${extractLecturesFromEventDescription(
+                event.description
+            ).concat(
+                `\n*Para encontrar el link a un paper podés hacerlo desde ${client.channels.cache.get(
+                    client.config.papersTextChannelID
+                )} o en la [web de la cátedra](https://algoritmos-iii.github.io/)*`
+            )}`,
+        },
+    ],
+    null,
+    null,
+    `${client.guilds.cache
+        .first()!
+        .roles.cache.get(client.config.studentRoleID)}`
```

Comment:
No es necesario el usar interpolacion de strings. En caso de que se quiera convertir a String se puede agregar un `.toString()` al final.

```suggestion
    client.guilds.cache
        .first()!
        .roles.cache.get(client.config.studentRoleID)
```

Reply:
Correcto, bien visto